Formula/y/yara-x.rb

```diff
@@ -15,10 +16,13 @@ class YaraX < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "dd8ec8bd9807a760e9d048969158301e2680168f1479865723ea1eb94db49215"
   end
 
+  depends_on "cargo-c" => :build
   depends_on "rust" => :build
 
   def install
     system "cargo", "install", *std_cargo_args(path: "cli")
+    system "cargo", "cinstall", "-p", "yara-x-capi", "--jobs", ENV.make_jobs.to_s, "--release",
```

Comment:
Should we use the `make_jobs` argument here? That seems specific to Makefiles

Reply:
The variable is not aptly named, it is the number of concurrent build jobs we can fire. It's not make-specific.